config/database.yml

```diff
@@ -1,25 +0,0 @@
-default: &default
```

Comment:
Has this config been deleted intentionally?

Reply:
No it probably shouldn't be deleted, I'll add it back in